itou/www/approvals_views/forms.py

```diff
@@ -83,11 +96,10 @@ def get_qs_filters(self):
             status_filters_list.append(Q(start_at__gt=now))
         if data.get("status_expired"):
             status_filters_list.append(Q(end_at__lt=now))
-        if status_filters_list:
-            status_filters = Q()
-            for status_filter in status_filters_list:
-                status_filters |= status_filter
-            qs_filters_list.append(status_filters)
+        qs_filters_list.append(Q(reduce(operator.or_, status_filters_list, Q())))
+
+        if expiry := int(data.get("expiry")):
```

Comment:
En mettant 
```suggestion
        if expiry := int(data.get("expiry", 0)):
```
ici, tu n'évites pas le besoin de mettre `data = {"expiry": "0"}` dans le `__init__` ?

Reply:
Je veux que le formulaire soit coché par défaut à Tous plutôt que d'avoir les 4 radio rouge pour indiquer que ça ne doit pas rester vide.

Reply:
Alternative: `self.form = ApprovalForm(self.siae.pk, self.request.GET or None, initial={"expiry": "0"})` dans la vue

Reply:
Oui, j'ai mis le `or None`, et la valeur initiale dans le field du formulaire plutôt.